app/controllers/users_controller.rb

```diff
@@ -280,7 +280,11 @@ def determine_follow_suggestions(current_user)
       attributes_to_select: INDEX_ATTRIBUTES_FOR_SERIALIZATION,
     ).suggest
 
-    recent_suggestions.presence || default_suggested_users
+    if SiteConfig.prefer_manual_suggested_users?
```

Comment:
Since we are allowing Forem creators to override follow suggestions, we can put this at the top of the method and short-circuit out if it is `true` (which would mean we wouldn't have to call out to the `Suggester::Users::Recent` if we don't need to 😄).

We also want to check that `default_suggested_users` is actually _set_ and has a value; otherwise, there could be a scenario where an admin checks the `prefer_manual_suggested_users` checkbox, but doesn't have any suggested users actually set!

```ruby
return default_suggested_users if SiteConfig.prefer_manual_suggested_users? && default_suggested_users

# recent_suggestions = ....
```

Reply:
Updated. Thank you!